com.unity.render-pipelines.core/Editor/Debugging/DebugWindow.cs

```diff
@@ -136,6 +137,13 @@ static void Init()
             open = true;
         }
 
+        [MenuItem("Window/Analysis/Rendering Debugger", validate = true)]
+        static bool ValidateMenuItem()
+        {
+            bool srpActive = RenderPipelineManager.currentPipeline != null;
+            return srpActive;
```

Comment:
Do we need the `srpActive` variable? Could it simply return `RenderPipelineManager.currentPipeline != null` ?